velox/type/fbhive/tests/HiveTypeSerializerTests.cpp

```diff
@@ -33,9 +33,19 @@ TEST(HiveTypeSerializer, primitive) {
   EXPECT_EQ(result, "bigint");
 }
 
+struct Foo {};
 TEST(HiveTypeSerializer, opaque) {
-  std::shared_ptr<const velox::Type> type = velox::OPAQUE<bool>();
+  // Use a custom name to highlight this is just an alias.
+  registerOpaqueType<Foo>("bar");
+
+  std::shared_ptr<const velox::Type> type = velox::OPAQUE<Foo>();
```

Comment:
you can omit the `velox::` namespace prefix throughout. You can also use the TypePtr alias or just auto